### PR TITLE
Add API Requests By User

### PR DESCRIPTION
**Pull request by larsxschneider**
PR with a few minor improvements for #148

### CODE REVIEW

File: updater/scripts/api-requests-by-user.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+#
+# Count number of API calls per user
+#
+
+echo -e "user\trequests"
+
+zcat -f /var/log/github/unicorn.log.1* |
+    grep -F 'request_category=api' |
+    grep -Fv 'remote_address=127.0.0.1' |
```
Comment (larsxschneider) [inline updater/scripts/api-requests-by-user.sh]: @dfarr I excluded internal API calls here as they are not directly issued by a user (e.g. Git LFS requests)

File: docs/housekeeping-api-requests.html [hunk 2]
```diff
@@ -51,6 +51,20 @@ <h3>API Requests</h3>
...
 
 <div class="chart-placeholder">
+	<h3>API Requests per User</h3>
+	<canvas
+		data-url="{{ site.dataURL }}/api-requests-by-user.tsv"
+		data-type="list"
+	></canvas>
+	<div class="info-box">
+		<p>
+			The top 20 users that made the highest number of API requests yesterday.
```
Comment (pluehne) [inline docs/housekeeping-api-requests.html]: “who” instead of ”that” might sound nicer.

File: updater/scripts/api-requests-by-user.sh (hunk 1 above)
Comment (dfarr) [reply updater/scripts/api-requests-by-user.sh]: Good idea 👍

**Comment by pluehne** [comment]
Thanks again, @dfarr!

**Comment by larsxschneider** [comment]
Thanks @dfarr from me too 👍 ! Please note, we just rebased the master branch. Somehow your authorship disappeared with the squash merge but I repaired: https://github.com/Autodesk/hubble/commit/b734a912c57f4475cd88ee393e4d4e46eb18c8b0

**Comment by larsxschneider** [comment]
9 contributors, yay! https://github.com/Autodesk/hubble/graphs/contributors

**Comment by pluehne** [comment]
That graph even shows 10 contributors now :wink:.

**Comment by dfarr** [comment]
Thanks guys! Always great to see your contributions in the graph :)

**Pull request merged**